index_ar.html

```diff
@@ -5,6 +5,10 @@
 direction: rtl
 
 pagecontent:
+  install:
+    install: Homebrew ثبّت
```

Comment:
> How did you get all these?

I am a master of all languages :wink:. *(I wish)*.

>  Why did they get moved?

It seemed more canonical given the new location of the install block on the page. I was possibly just being anal though.